stl/inc/filesystem

```diff
@@ -2969,45 +2969,35 @@ namespace filesystem {
         return {};
     }
 
-    _NODISCARD inline __std_win_error _Absolute(path& _Result, const wstring& _Text) { // pre: _Result.empty()
-        if (_Text.empty()) {
-            return __std_win_error::_Success;
-        }
-
-        _Result._Text.resize(__std_fs_max_path);
-        __std_win_error _Err;
-        for (;;) {
-            const auto _Requested_size = static_cast<unsigned long>(_Result._Text.size());
-            const auto _Full_path_result =
-                __std_fs_get_full_path_name(_Text.c_str(), _Requested_size, _Result._Text.data());
-            _Err = _Full_path_result._Error;
-            _Result._Text.resize(_Full_path_result._Size);
-            if (_Full_path_result._Size < _Requested_size) {
-                break;
-            }
-        }
-
-        return _Err;
-    }
-
-    _EXPORT_STD _NODISCARD inline path absolute(const path& _Input) {
+    _EXPORT_STD _NODISCARD inline path absolute(const path& _Input, error_code& _Ec) {

```

Comment:
I make `absolute(const path&, error_code&)` defined before `absolute(const path&)`. I notice that `...(...,error_code&)` is always defined after `...(...)` in the standard; however, as there are already a lot of examples in this order(`exists`, `read_symlink`, `create_directories` etc.), I think this change is ok.